packages/devextreme/scss/widgets/fluent/_sizes.scss

```diff
@@ -49,7 +49,7 @@ $fluent-button-icon-margin: null !default;
   $fluent-base-icon-size: 20px !default; // 24px !default
   $fluent-base-inline-horizontal-padding: 12px !default; // 10px !default;
   $fluent-base-border-radius: 4px !default;
-  $fluent-base-dropdown-widgets-shadow: 0 2px 7px $base-dropdown-shadow-color !default;
+  $fluent-base-dropdown-widgets-shadow: 0 0 2px 0 $base-dropdown-shadow-color1, 0 8px 16px 0 $base-dropdown-shadow-color2 !default;
```

Comment:
I did not find a task for changing the size of the shadow for dropdown. If this really needs to be done, then the relevant information should be in the card